lib/gds_api/content_register.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'base'
+require_relative 'exceptions'
+
+class GdsApi::ContentRegister < GdsApi::Base
+
+  def entries(format)
+    get_json(entries_url(format))
+  end
+
+  def entries!(format)
```

Comment:
I don't think we'd want both variants here.  For all other adapters that access an index endpoint we just have a non-bang method that calls `get_json!`.  It should be an exceptional case for the endpoint to return a 404.